lib/jekyll/admin.rb

```diff
@@ -1,7 +1,26 @@
 require "jekyll/admin/version"
-
+require "jekyll/admin/admin_servlet"
+require "jekyll/admin/api_servlet"
+require "jekyll/admin/api_handler"
+require "jekyll/admin/configuration/handler"
+require "jekyll/admin/configuration/servlet"
+require "jekyll/admin/pages/handler"
+require "jekyll/admin/pages/servlet"
```

Comment:
@benbalter autoload looks like a nice solution, but in my case most of the classes have the same name, i.e. Servlet and Handler. I am able to think of a way to make that work, can you help me out?